README.md

```diff
@@ -807,22 +807,22 @@ For the vast majority of cases this will get you most of the migration:
 
 ## Migrating from `react-rails` to `react_on_rails`
 
-### Why to migrate?
+### Why migrate
 
 [`react_on_rails`](https://github.com/shakacode/react_on_rails/) offers several additional features for a Rails + React application. The following is a table of features comparison.
 
-| **Feature**             | **react-rails** | **react-on-rails** |
-| ----------------------- |:---------------:|:------------------:|
-| Sprockets               | ✅               | ❌                  |
-| Shakapacker             | ✅               | ✅                  |
-| SSR                     | ✅               | ✅                  |
-| SSR with HMR            | ✅               | ✅                  |
-| SSR with React-Router   | ❌               | ✅                  |
-| SSR with Code Splitting | ❌               | ✅                  |
-| Node SSR                | ❌               | ✅                  |
-| Advanced Redux support  | ❌               | ✅                  |
-| ReScript support        | ❌               | ✅                  |
-| I18n support            | ❌               | ✅                  |
+| **Feature**             | **react-rails** | **react-on-rails** | **react-on-rails pro** |
+| ----------------------- |:---------------:|:------------------:|:----------------------:|
+| Sprockets               | ✅               | ❌                | ❌                     |
+| Shakapacker             | ✅               | ✅                | ✅                     |
+| SSR                     | ✅               | ✅                | ✅                     |
+| SSR with HMR            | ✅               | ❌                | ✅                     |
+| SSR with React-Router   | ❌               | ❌                | ✅                     |
+| SSR with Code Splitting | ❌               | ❌                | ✅                     |
+| Node SSR                | ❌               | ❌                | ✅                     |
+| Advanced Redux support  | ❌               | ❌                | ✅                     |
+| ReScript support        | ❌               | ❌                | ✅                     |
+| I18n support            | ❌               | ❌                | ✅                     |
```

Comment:
Are you sure about this?

In [react-webpack-rails-tutorial](https://github.com/shakacode/react-webpack-rails-tutorial/) we demonstrate many of these items use `react_on_rails`.

Reply:
Quoting the PR message:
> I'm not sure about what's free in react-on-rails. Can you confirm the table is right?

I only guessed from what's said in the website:
- https://www.shakacode.com/react-on-rails/docs/react-on-rails-pro/react-on-rails-pro/
- https://www.shakacode.com/react-on-rails-pro/

Reply:
Thanks for the clarification.

An older comparison between features offered by `react-rails` and `react_on_rails` can be found on `react_on_rails` official docs: [Shakapacker (Rails/Webpacker) React Integration Options](https://www.shakacode.com/react-on-rails/docs/guides/rails-webpacker-react-integration-options/)

I close the PR for now. You can update the PR with more solid references so we can review it again.

Thanks for your PR.